keps/sig-node/1769-memory-manager/README.md

```diff
@@ -778,3 +780,17 @@ The Memory Manager sets and enforces cgroup memory limit for ("on behalf of") a
 [hugepage-issue]: https://github.com/kubernetes/kubernetes/issues/80716
 [memory-issue]: https://github.com/kubernetes/kubernetes/issues/81009
 
+### Windows considerations
+
+Numa nodes can not be guaranteed via the Windows API, instead an [ideal Numa](https://learn.microsoft.com/en-us/windows/win32/procthread/numa-support#numa-support-on-systems-with-more-than-64-logical-processors) node can be 
```

Comment:
I am guessing we need to inform the user about this with a warning log message

Reply:
Not a bad idea to have a warning, I was thinking we need good docs on this topic.

Reply:
added some more details and note about docs/warning message